### PR TITLE
Release 0.12.2

### PR DESCRIPTION
**Pull request by jaelim-ms**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,19 @@
...
 
+## Version 0.12.2
+
+This version contains improvements. **Verified all functionalities against iOS 11 GM.**
+
+### MobileCenterCrashes
+
+* **[Improvement]** Added a millisecond precision to crash logs for more accurate log time.
+
+### MobileCenterDistribute
```
Comment (clpolet) [inline CHANGELOG.md]: Also the behavior has changed on iOS 11 now that we are using the new feature a popup is displayed.

Comment (clpolet) [reply CHANGELOG.md]: There was a workaround before and the behavior noticeably changed between the two versions.

Comment (jaelim-ms) [reply CHANGELOG.md]: OK. We are bringing it back so I added that to the change log.

Comment (jaelim-ms) [reply CHANGELOG.md]: For popup thing, it is a new behavior on iOS 11. Not mentioning details here.

Comment (TroubleMakerBen) [reply CHANGELOG.md]: Maybe say "When authenticating for in-app-updates, iOS 11 now displays a confirmation dialog to access login credentials"? I would mention it.

**Pull request merged**